packages/dashboard/src/components/Card/GridList/GridList.tsx

```diff
@@ -44,6 +48,15 @@ export class GridList extends React.Component<IGridListProps> {
     );
   }
 
+  private _onRenderRow(props: IDetailsRowProps): JSX.Element {
+    const styles: Partial<IDetailsRowStyles> = {
+      cell: {
+        paddingLeft: '0px'
+      }
+    };
+    return <DetailsRow {...props} styles={styles} />;
+  }
+
   // Disabling ts-lint for the signature because Details List expects row parameter as any
```

Comment:
why do we have two code reviews for different bugs that have the same code? Please consolidate and send one